README.md

```diff
@@ -6,168 +6,171 @@
-* Open up the environment for the discussion through integrated Discussion platform
-* Act with Open Stat
-* Appropriate Dashboard with most APIs Activities log of projects and other things.
-* Sourced Packaged, Custom made software ( can be changed according to needs after downloading it.
-* Provides various opportunities to integrate many other Open Source projects recognized by organizations
+# Setting up the donut frontend potion locally.
 
+**Forking The Repository**
 
+The first step of setting the donut frontend locally is to fork the original repository. This creates a copy of that repository under your own account enabling you to begin working with the code. The rights to public repositories will be such that you can view the code, but not directly commit into the repository or create branches. This allows the project owners to control changes within their codebase.
 
-Installation
-------------
+The forking step creates a copy to which you do have permission to commit and branch on and you can consider this your working copy of the project. You can make changes and commits here, safe in the knowledge that you will not affect the main repository.
 
-[![NODE JS](http://img.shields.io/badge/NodeJS-required-teal.svg?logo=node.js)](https://nodejs.org/en/) [![NPM](http://img.shields.io/badge/npm-required-red.svg?logo=npm)](http://npmjs.com/)
+In order to fork the donut frontend repository visit [here](https://github.com/codeuino/social-platform-donut-frontend)
 
-To get this project running locally on your machine follows the steps below in order:
+![](https://i.imgur.com/bWLML1u.png)
 
-1. fork this repo
-2. clone the forked repo from your GitHub 
-using the command `git clone https://github.com/username/Social-Platform-Donut.git` don't forget to replace the username in the URL
+**Cloning the forked repository**
 
-    ![fork-and-clone](READMEassets/forking-the-repo.gif)
+In order to clone the forked repository visit fork of the donut repository under "your repositories" section and click on the “Clone or download” which will open a UI showing the Git URL. A button to the right hand side of the URL allows you to copy it into your clipboard.
 
-3. Checkout to the development branch
+![](https://i.imgur.com/2UyN3KV.png)
+There are various graphical tools you can use to work with Git repositories but for simple procedures, the command line is often fastest.
 
-    ```shell
-        git checkout development
-    ```
+Open a command window and navigate to the path where you would like to clone the repository.
+Use the following command to begin a clone:
 
-4. Setting up the project locally.
+```
+git clone https://github.com/<your github username>/social-platform-donut-frontend.git
+```
 
-    ```shell
-    npm install
-    ```
+Once the command completes you will have a new folder containing the cloned repository. We can validate this by running the “dir” command. We can then move into that directory using the "cd " command.
```

Comment:
The above command for clone will create "social-platform-donut-frontend" so I think you can change it to ```cd social-platform-donut-frontend```